frontend_agent_beginning

Pragya1805 opened the pull request.

Pragya1805 committed:
frontend_agent_beginning

trblessy commented on frontend/gpay_agent/lib/main.dart:
```diff
@@ -0,0 +1,22 @@
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
```
Please give a more descriptive name instead of MyApp

trblessy commented on frontend/gpay_agent/lib/views/fetch_store.dart:
```diff
@@ -0,0 +1,133 @@
+                  height: 50,
+                  child: RaisedButton(
+                    onPressed: () {
+                      print(num);
```
Please remove any debug statements

trblessy commented on frontend/gpay_agent/lib/views/login_view.dart:
```diff
@@ -0,0 +1,82 @@
+}
+
+class _LoginViewState extends State<LoginView>{
+  GoogleSignIn _googleSignIn = GoogleSignIn(scopes: ['email']);
```
Add a class level comment describing the class.
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-a-library-level-doc-comment

trblessy commented on frontend/gpay_agent/lib/views/qr_scan.dart:
```diff
@@ -0,0 +1,101 @@
+  Future _scan() async {
+    String barcode = await scanner.scan();
+    print(barcode);
+    // use bar code to get merchant, if merchant present, pass merchant name, fou,d, else not found
```
typo : found

trblessy replied:
https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences

trblessy commented on frontend/gpay_agent/lib/views/welcome_agent.dart:
```diff
@@ -0,0 +1,87 @@
+import 'package:google_sign_in/google_sign_in.dart';
+
+
+
```
remove extra newlines

trblessy reviewed (commented):
By convention, the description of the PR is a complete sentence, written as though it were an order
'Add dart files for registration views of the Gpay merchant app'

followed by more description of the exact changes

Pragya1805 committed:
added comments, deleted debug statements, used descriptive name(following style guides), corrected typos, removed extra newlines

Pragya1805 commented:
Please review. I've made some changes - added comments, deleted debug statements, used descriptive name(following style guides), corrected typos, removed extra newlines.

trblessy commented on frontend/gpay_agent/lib/main.dart:
```diff
@@ -0,0 +1,22 @@
+  runApp(AgentApp());
+}
+
+///Defines the the theme of application and directs to [LoginView] interface.
```
space between /// and the comment, here and in other files

trblessy commented on frontend/gpay_agent/lib/views/fetch_store.dart:
```diff
@@ -0,0 +1,116 @@
+                  ),
+                ),
+                ///Fetches store details using phone number of store.
+                ///If store number is regestered in databsase, it calls [MerchantFound]
```
database

Pragya1805 committed:
spaces added between /// and comment, corrected typos
